guides/doc-Upgrading_and_Updating/topics/updating_satellite_server_to_next_minor_version.adoc

```diff
@@ -7,7 +7,7 @@
 
 * Ensure that you have synchronized {ProjectServer} repositories for {Project}, {SmartProxy}, and {project-client-name}.
 * Ensure each external {SmartProxy} and Content Host can be updated by promoting the updated repositories to all relevant Content Views.
-
+~/{Project}-repos
```

Comment:
This looks like an odd thing. Copy paste error?

Reply:
yeh correcting